modules/ocm-networking-tab-adding-ingress.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// ocm/ocm-overview.adoc
+:_content-type: PROCEDURE
+[id="ocm-networking-tab-adding-ingress_{context}"]
+= Adding a network ingress to your {product-title} cluster
+
+You can add a network ingress to your cluster from the {cluster-manager-url} web UI.
+
+.Prerequisites
+
+* You have a Red Hat account.
+* You have the required permissions to make changes to your cluster in {cluster-manager}.
+
+.Procedure
+
+. From the **Networking** tab in {cluster-manager}, click the "Additional application router" toggle to enable the ingress. There are two options you can add to the additional router:
```

Comment:
Is "Additional application router" a GUI button? Would it be in asterisks? (*)

Reply:
Does "the ingress" need a noun after it? For example, "the ingress route," or "the ingress port," or "the ingress load balancer"? That said, there are a very few instances of "the Ingress" popping up (capitalized). Ingress is always capitalized on the Kubernetes website (https://kubernetes.io/docs/concepts/services-networking/ingress/), except in rare cases (ingress-nginix).

I think "the Ingress" is ok.